<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Provides access to smokingMouse project data • smokingMouse</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><link href="extra.css" rel="stylesheet">
<meta property="og:title" content="Provides access to smokingMouse project data">
<meta property="og:description" content="This is an ExperimentHub package that provides access to the data generated and analyzed in the [smoking-nicotine-mouse](https://github.com/LieberInstitute/smoking-nicotine-mouse/) LIBD project. The datasets contain the expression data of mouse genes, transcripts, exons, and exon-exon junctions across 208 samples from pup and adult mouse brain, and adult blood, that were exposed to nicotine, cigarette smoke, or controls. They also contain relevant metadata of these samples and gene expression features, such QC metrics, if they were used after filtering steps and also if the features were differently expressed in the different experiments.">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">smokingMouse</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.5.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="articles/smokingMouse.html">Get started</a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
<li>
  <a href="news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/LieberInstitute/smokingMouse/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><div class="row">
  <div class="contents col-md-9">
<div class="section level1">
<div class="page-header"><h1 id="smokingmouse">smokingMouse<a class="anchor" aria-label="anchor" href="#smokingmouse"></a>
</h1></div>
<!-- badges: start -->

<!-- badges: end -->
<p>Welcome to the <code>smokingMouse</code> project! Here you’ll be able to access the mouse expression data used for the analysis of the smoking-nicotine-mouse LIBD project.</p>
<div class="section level2">
<h2 id="overview">Overview<a class="anchor" aria-label="anchor" href="#overview"></a>
</h2>
<p>This bulk RNA-sequencing project consisted of a differential expression analysis (DEA) involving 4 data types: genes, transcripts, exons, and exon-exon junctions. The main goal of this study was to explore the effects of prenatal exposure to smoking and nicotine on the developing mouse brain. As secondary objectives, this work evaluated: 1) the affected genes by each exposure in the adult female brain in order to compare offspring and adult results, and 2) the effects of smoking on adult blood and brain to search for overlapping biomarkers in both tissues. Finally, DEGs identified in mouse were compared against previously published results in human (Semick et al., 2020 and Toikumo et al., 2023).</p>
</div>
<div class="section level2">
<h2 id="study-design">Study design<a class="anchor" aria-label="anchor" href="#study-design"></a>
</h2>
<figure><img src="reference/figures/Study_design.png" align="center" width="800px"><figcaption style="color: gray; line-height: 0.88; text-align: justify"><p><font size="-1.5"><b>Experimental design of the study</b>. <b>A)</b> 21 pregnant mice were split into two experiments: in the first one prenatal nicotine exposure (PNE) was modeled administering nicotine (n=3) or vehicle (n=3) to the dams during gestation, and in the second maternal smoking during pregnancy (MSDP) was modeled exposing dams to cigarette smoke during gestation (n=8) or using them as controls (n=7). A total of 137 pups were born: 19 were born to nicotine-administered mice, 23 to vehicle-administered mice, 46 to smoking-exposed mice, and 49 to smoking control mice. 17 nonpregnant adult females were also nicotine-administered (n=9) or vehicle-administered (n=8) to model adult nicotine exposure, and 9 additional nonpregnant dams were smoking-exposed (n=4) or controls (n=5) to model adult smoking. Frontal cortex samples of all P0 pups (n=137: 42 for PNE and 95 for MSDP) and adults (n=47: 23 for the nicotine experiment and 24 for the smoking experiment) were obtained, as well as blood samples from the smoking-exposed and smoking control adults (n=24), totaling 208 samples. Number of donors and samples are indicated in the figure. <b>B)</b> RNA was extracted from such samples and bulk RNA-seq experiments were performed, obtaining expression counts for genes, transcripts, exons, and exon-exon junctions.</font></p>
<p></p>
</figcaption></figure>
</div>
<div class="section level2">
<h2 id="workflow">Workflow<a class="anchor" aria-label="anchor" href="#workflow"></a>
</h2>
<p>The next table summarizes the analyses done at each level.</p>
<figure><img src="reference/figures/Table_of_Analyses.png" align="center" width="800px"><figcaption style="color: gray; line-height: 0.94; text-align: justify"><p><font size="-1.5"><b> Summary of analysis steps across gene expression feature levels</b>:</font></p>
<p><b>1. Data processing</b>: counts of genes, exons, and exon-exon junctions were normalized to CPM and log2-transformed; transcript expression values were only log2-transformed since they were already in TPM. Lowly-expressed features were removed using the indicated functions and samples were separated by tissue and age in order to create subsets of the data for downstream analyses.</p>
<p><b>2. Exploratory Data Analysis (EDA)</b>: QC metrics of the samples were examined and used to filter the poor quality ones. Sample level effects were explored through dimensionality reduction methods and segregated samples in PCA plots were removed from the datasets. Gene level effects were evaluated with analyses of variance partition.</p>
<p><b>3. Differential Expression Analysis (DEA)</b>: with the relevant variables identified in the previous steps, the DEA was performed at the gene level for nicotine and smoking exposure in adult and pup brain samples, and for smoking exposure in adult blood samples; DEA at the rest of the levels was performed for both exposures in pup brain only. DE signals of the genes in the different conditions, ages, tissues, and species (using human results from <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msup><mi></mi><mn>1</mn></msup><annotation encoding="application/x-tex">^1</annotation></semantics></math><a href="https://www.nature.com/articles/s41380-018-0223-1" class="external-link">Semick et al., 2020</a>) were contrasted, as well as the DE signals of exons and transcripts vs those of their genes. Mean expression of DEGs and non-DEGs genes with and without DE features was also analyzed. Then, all resultant DEGs and DE features (and their genes) were compared by direction of regulation (up or down) between and within exposures (nicotine/smoking); mouse DEGs were also compared against human genes associated with TUD from <math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><msup><mi></mi><mn>2</mn></msup><annotation encoding="application/x-tex">^2</annotation></semantics></math><a href="https://www.medrxiv.org/content/10.1101/2023.03.27.23287713v2" class="external-link">Toikumo et al., 2023</a>.</p>
<p><b>4. Functional Enrichment Analysis</b>: GO &amp; KEGG terms significantly enriched in the clusters of DEGs and genes of DE transcripts and exons were obtained.</p>
<p><b>5. DGE visualization</b>: the log2-normalized expression of DEGs was represented in heat maps in order to distinguish the groups of up- and down-regulated genes.</p>
<p><b>6. Novel junction gene annotation</b>: for uncharacterized DE junctions with no annotated gene, their nearest, preceding, and following genes were determined. </p>
<p><font size="0.8">Abbreviations: Jxn: junction; Tx(s): transcript(s); CPM: counts per million; TPM: transcripts per million; TMM: Trimmed Mean of M-Values; TMMwsp: TMM with singleton pairing; QC: quality control; PC: principal component; DEA: differential expression analysis; DE: differential expression/differentially expressed; FC: fold-change; FDR: false discovery rate; DEGs: differentially expressed genes; TUD: tobacco use disorder; DGE: differential gene expression. </font></p>
</figcaption></figure><p>All <code>R</code> scripts created to perform such analyses can be found in <a href="https://github.com/LieberInstitute/smoking-nicotine-mouse/" class="external-link">code on GitHub</a>.</p>
</div>
<div class="section level2">
<h2 id="smoking-mouse-datasets">smoking Mouse datasets<a class="anchor" aria-label="anchor" href="#smoking-mouse-datasets"></a>
</h2>
<p>The mouse datasets contain the following data in a single <code>R</code> <code>RangedSummarizedExperiment</code>* object for each feature (genes, transcripts, exons, and exon-exon junctions):</p>
<ul>
<li>
<strong>Raw data</strong>: raw read counts (for genes, exons, and junctions) or TPM (for transcripts), also including the original metadata of the expression features and samples.</li>
<li>
<strong>Processed data</strong>: normalized and log-scaled counts of the same features (log(CPM+0.5) for genes, exons, and junctions, or log(TPM+0.5) for transcripts). In addition to the feature and sample information, the datasets contain information of which ones were used in downstream analyses (the ones that passed filtering steps), and which features were differentially expressed in the different experiments.</li>
</ul>
<p>Moreover, you can find human data generated in Semick et al., (2018) in Mol Psychiatry (DOI: <a href="https://doi.org/10.1038/s41380-018-0223-1" class="external-link uri">https://doi.org/10.1038/s41380-018-0223-1</a>) that contain the results of a DEA for cigarette smoke exposure in adult and prenatal human brain.</p>
<p>*For more details, check the documentation for <a href="https://www.bioconductor.org/packages/devel/bioc/vignettes/SummarizedExperiment/inst/doc/SummarizedExperiment.html" class="external-link"><code>RangedSummarizedExperiment</code></a> objects.</p>
</div>
<div class="section level2">
<h2 id="data-specifics">Data specifics<a class="anchor" aria-label="anchor" href="#data-specifics"></a>
</h2>
<ul>
<li>
<em>‘rse_gene_mouse_RNAseq_nic-smo.Rdata’</em>: (<code>rse_gene</code> object) the gene RSE object contains the raw and log-normalized expression data of 55,401 mouse genes across the 208 samples from brain and blood of control and nicotine/smoking-exposed pup and adult mice.</li>
<li>
<em>‘rse_tx_mouse_RNAseq_nic-smo.Rdata’</em>: (<code>rse_tx</code> object) the tx RSE object contains the raw and log-scaled expression data of 142,604 mouse transcripts across the 208 samples from brain and blood of control and nicotine/smoking-exposed pup and adult mice.</li>
<li>
<em>‘rse_exon_mouse_RNAseq_nic-smo.Rdata’</em>: (<code>rse_exon</code> object) the exon RSE object contains the raw and log-normalized expression data of 447,670 mouse exons across the 208 samples from brain and blood of control and nicotine/smoking-exposed pup and adult mice.</li>
<li>
<em>‘rse_jx_mouse_RNAseq_nic-smo.Rdata’</em>: (<code>rse_jx</code> object) the jx RSE object contains the raw and log-normalized expression data of 1,436,068 mouse exon-exon junctions across the 208 samples from brain and blood of control and nicotine/smoking-exposed pup and adult mice.</li>
</ul>
<p>All the above datasets contain the sample and feature metadata and additional data of the results obtained in the filtering steps and the DEA.</p>
<ul>
<li>
<em>‘de_genes_prenatal_human_brain_smoking.Rdata’</em>: (<code>de_genes_prenatal_human_brain_smoking</code> object) data frame with DE statistics of 18,067 human genes for cigarette smoke exposure in prenatal human cortical tissue.</li>
<li>
<em>‘de_genes_adult_human_brain_smoking.Rdata’</em>: (<code>de_genes_adult_human_brain_smoking</code> object) data frame with DE statistics of 18,067 human genes for cigarette smoke exposure in adult human cortical tissue.</li>
</ul>
</div>
<div class="section level2">
<h2 id="installation-instructions">Installation instructions<a class="anchor" aria-label="anchor" href="#installation-instructions"></a>
</h2>
<p>Get the latest stable <code>R</code> release from <a href="http://cran.r-project.org/" class="external-link">CRAN</a>. Then install <code>smokingMouse</code> from <a href="http://bioconductor.org/" class="external-link">Bioconductor</a> using the following code:</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html" class="external-link">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span> <span class="op">{</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span><span class="op">}</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"smokingMouse"</span><span class="op">)</span></span></code></pre></div>
<p>And the development version from <a href="https://github.com/LieberInstitute/smokingMouse" class="external-link">GitHub</a> with:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"LieberInstitute/smokingMouse"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="example-of-how-to-access-the-data">Example of how to access the data<a class="anchor" aria-label="anchor" href="#example-of-how-to-access-the-data"></a>
</h2>
<p>Below there’s example code on how to access the mouse and human gene data but can do the same for any of the datasets previously described. The datasets are retrieved from <a href="http://bioconductor.org/" class="external-link">Bioconductor</a> <code>ExperimentHub</code>.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Connect to ExperimentHub</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">ExperimentHub</span><span class="op">)</span></span>
<span><span class="co">#&gt; Loading required package: BiocGenerics</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Attaching package: 'BiocGenerics'</span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:stats':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     IQR, mad, sd, var, xtabs</span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:base':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     anyDuplicated, aperm, append, as.data.frame, basename, cbind,</span></span>
<span><span class="co">#&gt;     colnames, dirname, do.call, duplicated, eval, evalq, Filter, Find,</span></span>
<span><span class="co">#&gt;     get, grep, grepl, intersect, is.unsorted, lapply, Map, mapply,</span></span>
<span><span class="co">#&gt;     match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,</span></span>
<span><span class="co">#&gt;     Position, rank, rbind, Reduce, rownames, sapply, setdiff, table,</span></span>
<span><span class="co">#&gt;     tapply, union, unique, unsplit, which.max, which.min</span></span>
<span><span class="co">#&gt; Loading required package: AnnotationHub</span></span>
<span><span class="co">#&gt; Loading required package: BiocFileCache</span></span>
<span><span class="co">#&gt; Loading required package: dbplyr</span></span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">eh</span> <span class="op">&lt;-</span> <span class="fu">ExperimentHub</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/ExperimentHub/man/ExperimentHub-class.html" class="external-link">ExperimentHub</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span> </span>
<span><span class="co">## Load the datasets of the package</span></span>
<span><span class="va">myfiles</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/AnnotationHub/man/AnnotationHub-class.html" class="external-link">query</a></span><span class="op">(</span><span class="va">eh</span>, <span class="st">"smokingMouse"</span><span class="op">)</span></span>
<span>  </span>
<span><span class="co">########################</span></span>
<span><span class="co">#      Mouse data </span></span>
<span><span class="co">########################</span></span>
<span><span class="co">## Download the mouse gene data</span></span>
<span><span class="va">rse_gene</span> <span class="op">&lt;-</span> <span class="va">myfiles</span><span class="op">[[</span><span class="st">'EH8313'</span><span class="op">]</span><span class="op">]</span> </span>
<span><span class="co">#&gt; Warning: package 'GenomicRanges' was built under R version 4.4.1</span></span></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## This is a RangedSummarizedExperiment object</span></span>
<span><span class="va">rse_gene</span></span>
<span><span class="co">#&gt; class: RangedSummarizedExperiment </span></span>
<span><span class="co">#&gt; dim: 55401 208 </span></span>
<span><span class="co">#&gt; metadata(1): Obtained_from</span></span>
<span><span class="co">#&gt; assays(2): counts logcounts</span></span>
<span><span class="co">#&gt; rownames(55401): ENSMUSG00000102693.1 ENSMUSG00000064842.1 ...</span></span>
<span><span class="co">#&gt;   ENSMUSG00000064371.1 ENSMUSG00000064372.1</span></span>
<span><span class="co">#&gt; rowData names(13): Length gencodeID ... DE_in_pup_brain_nicotine</span></span>
<span><span class="co">#&gt;   DE_in_pup_brain_smoking</span></span>
<span><span class="co">#&gt; colnames: NULL</span></span>
<span><span class="co">#&gt; colData names(71): SAMPLE_ID FQCbasicStats ...</span></span>
<span><span class="co">#&gt;   retained_after_QC_sample_filtering</span></span>
<span><span class="co">#&gt;   retained_after_manual_sample_filtering</span></span></code></pre></div>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## Check sample info </span></span>
<span><span class="fu">colData</span><span class="op">(</span><span class="va">rse_gene</span><span class="op">)</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">5</span>, <span class="fl">1</span><span class="op">:</span><span class="fl">5</span><span class="op">]</span></span>
<span><span class="co">#&gt; DataFrame with 5 rows and 5 columns</span></span>
<span><span class="co">#&gt;     SAMPLE_ID FQCbasicStats perBaseQual perTileQual  perSeqQual</span></span>
<span><span class="co">#&gt;   &lt;character&gt;   &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt;</span></span>
<span><span class="co">#&gt; 1 Sample_2914          PASS        PASS        PASS        PASS</span></span>
<span><span class="co">#&gt; 2 Sample_4042          PASS        PASS        PASS        PASS</span></span>
<span><span class="co">#&gt; 3 Sample_4043          PASS        PASS        PASS        PASS</span></span>
<span><span class="co">#&gt; 4 Sample_4044          PASS        PASS        PASS        PASS</span></span>
<span><span class="co">#&gt; 5 Sample_4045          PASS        PASS        PASS        PASS</span></span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Check gene info</span></span>
<span><span class="fu">rowData</span><span class="op">(</span><span class="va">rse_gene</span><span class="op">)</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">5</span>, <span class="fl">1</span><span class="op">:</span><span class="fl">5</span><span class="op">]</span></span>
<span><span class="co">#&gt; DataFrame with 5 rows and 5 columns</span></span>
<span><span class="co">#&gt;                         Length            gencodeID          ensemblID</span></span>
<span><span class="co">#&gt;                      &lt;integer&gt;          &lt;character&gt;        &lt;character&gt;</span></span>
<span><span class="co">#&gt; ENSMUSG00000102693.1      1070 ENSMUSG00000102693.1 ENSMUSG00000102693</span></span>
<span><span class="co">#&gt; ENSMUSG00000064842.1       110 ENSMUSG00000064842.1 ENSMUSG00000064842</span></span>
<span><span class="co">#&gt; ENSMUSG00000051951.5      6094 ENSMUSG00000051951.5 ENSMUSG00000051951</span></span>
<span><span class="co">#&gt; ENSMUSG00000102851.1       480 ENSMUSG00000102851.1 ENSMUSG00000102851</span></span>
<span><span class="co">#&gt; ENSMUSG00000103377.1      2819 ENSMUSG00000103377.1 ENSMUSG00000103377</span></span>
<span><span class="co">#&gt;                                 gene_type    EntrezID</span></span>
<span><span class="co">#&gt;                               &lt;character&gt; &lt;character&gt;</span></span>
<span><span class="co">#&gt; ENSMUSG00000102693.1                  TEC       71042</span></span>
<span><span class="co">#&gt; ENSMUSG00000064842.1                snRNA          NA</span></span>
<span><span class="co">#&gt; ENSMUSG00000051951.5       protein_coding      497097</span></span>
<span><span class="co">#&gt; ENSMUSG00000102851.1 processed_pseudogene   100418032</span></span>
<span><span class="co">#&gt; ENSMUSG00000103377.1                  TEC          NA</span></span></code></pre></div>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">## Access the original counts</span></span>
<span><span class="va">original_counts</span> <span class="op">&lt;-</span> <span class="fu">assays</span><span class="op">(</span><span class="va">rse_gene</span><span class="op">)</span><span class="op">$</span><span class="va">counts</span></span>
<span><span class="co">## Access the log-normalized counts</span></span>
<span><span class="va">logcounts</span> <span class="op">&lt;-</span> <span class="fu">assays</span><span class="op">(</span><span class="va">rse_gene</span><span class="op">)</span><span class="op">$</span><span class="va">logcounts</span></span>
<span></span>
<span></span>
<span><span class="co">########################</span></span>
<span><span class="co">#      Human data </span></span>
<span><span class="co">########################</span></span>
<span><span class="co">## Download the human gene data</span></span>
<span><span class="va">de_genes_prenatal_human_brain_smoking</span> <span class="op">&lt;-</span> <span class="va">myfiles</span><span class="op">[[</span><span class="st">'EH8317'</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="co">## This is a data frame</span></span>
<span><span class="va">de_genes_prenatal_human_brain_smoking</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">5</span>, <span class="op">]</span></span>
<span><span class="co">#&gt; GRanges object with 5 ranges and 9 metadata columns:</span></span>
<span><span class="co">#&gt;                   seqnames              ranges strand |    Length      Symbol</span></span>
<span><span class="co">#&gt;                      &lt;Rle&gt;           &lt;IRanges&gt;  &lt;Rle&gt; | &lt;integer&gt; &lt;character&gt;</span></span>
<span><span class="co">#&gt;   ENSG00000080709     chr5 113696642-113832337      + |      3995       KCNN2</span></span>
<span><span class="co">#&gt;   ENSG00000070886     chr1   22890057-22930087      + |      5358       EPHA8</span></span>
<span><span class="co">#&gt;   ENSG00000218336     chr4 183065140-183724177      + |     11983       TENM3</span></span>
<span><span class="co">#&gt;   ENSG00000189108     chrX 103810996-105011822      + |      3146    IL1RAPL2</span></span>
<span><span class="co">#&gt;   ENSG00000186732    chr22   43807202-43903728      + |      5821      MPPED1</span></span>
<span><span class="co">#&gt;                    EntrezID     logFC   AveExpr         t     P.Value adj.P.Val</span></span>
<span><span class="co">#&gt;                   &lt;integer&gt; &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;   &lt;numeric&gt; &lt;numeric&gt;</span></span>
<span><span class="co">#&gt;   ENSG00000080709      3781 -0.694069   2.86444  -6.09779 2.59861e-06 0.0469491</span></span>
<span><span class="co">#&gt;   ENSG00000070886      2046  1.545861   1.58351   5.67106 7.49034e-06 0.0477263</span></span>
<span><span class="co">#&gt;   ENSG00000218336     55714  0.804367   6.31125   5.55661 9.97733e-06 0.0477263</span></span>
<span><span class="co">#&gt;   ENSG00000189108     26280 -1.035988   1.62624  -5.53375 1.05665e-05 0.0477263</span></span>
<span><span class="co">#&gt;   ENSG00000186732       758  0.384536   9.34706   5.41518 1.42396e-05 0.0514535</span></span>
<span><span class="co">#&gt;                           B</span></span>
<span><span class="co">#&gt;                   &lt;numeric&gt;</span></span>
<span><span class="co">#&gt;   ENSG00000080709   4.44638</span></span>
<span><span class="co">#&gt;   ENSG00000070886   3.18385</span></span>
<span><span class="co">#&gt;   ENSG00000218336   3.53830</span></span>
<span><span class="co">#&gt;   ENSG00000189108   2.83949</span></span>
<span><span class="co">#&gt;   ENSG00000186732   3.19865</span></span>
<span><span class="co">#&gt;   -------</span></span>
<span><span class="co">#&gt;   seqinfo: 25 sequences from an unspecified genome; no seqlengths</span></span></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span></span>
<span><span class="co">## Access data of human genes as normally do with data frames</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="citation">Citation<a class="anchor" aria-label="anchor" href="#citation"></a>
</h2>
<p>Below is the citation output from using <code>citation('smokingMouse')</code> in R. Please run this yourself to check for any updates on how to cite <strong>smokingMouse</strong>.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/utils/citation.html" class="external-link">citation</a></span><span class="op">(</span><span class="st">'smokingMouse'</span><span class="op">)</span>, bibtex <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="co">#&gt; To cite package 'smokingMouse' in publications use:</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   Gonzalez-Padilla D, Collado-Torres L (2024). _Provides access to</span></span>
<span><span class="co">#&gt;   smokingMouse project data_. doi:10.18129/B9.bioc.smokingMouse</span></span>
<span><span class="co">#&gt;   &lt;https://doi.org/10.18129/B9.bioc.smokingMouse&gt;,</span></span>
<span><span class="co">#&gt;   https://github.com/LieberInstitute/smokingMouse/smokingMouse - R</span></span>
<span><span class="co">#&gt;   package version 1.3.0,</span></span>
<span><span class="co">#&gt;   &lt;http://www.bioconductor.org/packages/smokingMouse&gt;.</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; A BibTeX entry for LaTeX users is</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;   @Manual{,</span></span>
<span><span class="co">#&gt;     title = {Provides access to smokingMouse project data},</span></span>
<span><span class="co">#&gt;     author = {Daianna Gonzalez-Padilla and Leonardo Collado-Torres},</span></span>
<span><span class="co">#&gt;     year = {2024},</span></span>
<span><span class="co">#&gt;     url = {http://www.bioconductor.org/packages/smokingMouse},</span></span>
<span><span class="co">#&gt;     note = {https://github.com/LieberInstitute/smokingMouse/smokingMouse - R package version 1.3.0},</span></span>
<span><span class="co">#&gt;     doi = {10.18129/B9.bioc.smokingMouse},</span></span>
<span><span class="co">#&gt;   }</span></span></code></pre></div>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; To cite the original smoking-nicotine mouse work please use: </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; (TODO)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; To cite the original work from which human data come please use the following citation:</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Semick, S. A., Collado-Torres, L., Markunas, C. A., Shin, J. H., Deep-Soboslay, A., Tao, R., ... </span></span>
<span><span class="co">#&gt; &amp; Jaffe, A. E. (2020). Developmental effects of maternal smoking during pregnancy on the human</span></span>
<span><span class="co">#&gt;  frontal cortex transcriptome. Molecular psychiatry, 25(12), 3267-3277.</span></span>
<span><span class="co">#&gt; </span></span></code></pre></div>
<p>Please note that the <code>smokingMouse</code> package and the <a href="https://github.com/LieberInstitute/smoking-nicotine-mouse/" class="external-link">study analyses</a> were only made possible thanks to many other <code>R</code> and bioinformatics software authors, which are cited either in the vignette and/or the paper describing this study.</p>
</div>
<div class="section level2">
<h2 id="code-of-conduct">Code of Conduct<a class="anchor" aria-label="anchor" href="#code-of-conduct"></a>
</h2>
<p>Please note that the <code>smokingMouse</code> project is released with a <a href="http://bioconductor.org/about/code-of-conduct/" class="external-link">Contributor Code of Conduct</a>. By contributing to this project, you agree to abide by its terms.</p>
</div>
<div class="section level2">
<h2 id="development-tools">Development tools<a class="anchor" aria-label="anchor" href="#development-tools"></a>
</h2>
<ul>
<li>Continuous code testing is possible thanks to <a href="https://www.tidyverse.org/blog/2020/04/usethis-1-6-0/" class="external-link">GitHub actions</a> through <em><a href="https://CRAN.R-project.org/package=usethis" class="external-link">usethis</a></em>, <em><a href="https://CRAN.R-project.org/package=remotes" class="external-link">remotes</a></em>, and <em><a href="https://CRAN.R-project.org/package=rcmdcheck" class="external-link">rcmdcheck</a></em> customized to use <a href="https://www.bioconductor.org/help/docker/" class="external-link">Bioconductor’s docker containers</a> and <em><a href="https://bioconductor.org/packages/3.20/BiocCheck" class="external-link">BiocCheck</a></em>.</li>
<li>Code coverage assessment is possible thanks to <a href="https://codecov.io/gh" class="external-link">codecov</a> and <em><a href="https://CRAN.R-project.org/package=covr" class="external-link">covr</a></em>.</li>
<li>The <a href="http://LieberInstitute.github.io/smokingMouse" class="external-link">documentation website</a> is automatically updated thanks to <em><a href="https://CRAN.R-project.org/package=pkgdown" class="external-link">pkgdown</a></em>.</li>
<li>The code is styled automatically thanks to <em><a href="https://CRAN.R-project.org/package=styler" class="external-link">styler</a></em>.</li>
<li>The documentation is formatted thanks to <em><a href="https://CRAN.R-project.org/package=devtools" class="external-link">devtools</a></em> and <em><a href="https://CRAN.R-project.org/package=roxygen2" class="external-link">roxygen2</a></em>.</li>
</ul>
<p>For more details, check the <code>dev</code> directory.</p>
<p>This package was developed using <em><a href="https://bioconductor.org/packages/3.20/biocthis" class="external-link">biocthis</a></em>.</p>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2 data-toc-skip>Links</h2>
<ul class="list-unstyled">
<li><a href="https://www.bioconductor.org/packages/smokingMouse" class="external-link">View on Bioconductor</a></li>
<li><a href="https://github.com/LieberInstitute/smokingMouse/" class="external-link">Browse source code</a></li>
<li><a href="https://support.bioconductor.org/t/smokingMouse" class="external-link">Report a bug</a></li>
</ul>
</div>

<div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="https://www.r-project.org/Licenses/Artistic-2.0" class="external-link">Artistic-2.0</a></li>
</ul>
</div>

<div class="community">
<h2 data-toc-skip>Community</h2>
<ul class="list-unstyled">
<li><a href="CONTRIBUTING.html">Contributing guide</a></li>
<li><a href="CODE_OF_CONDUCT.html">Code of conduct</a></li>
<li><a href="SUPPORT.html">Getting help</a></li>
</ul>
</div>

<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing smokingMouse</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Daianna Gonzalez-Padilla <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0009-0005-8348-3195" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>

<div class="dev-status">
<h2 data-toc-skip>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/LieberInstitute/smokingMouse/issues" class="external-link"><img src="https://img.shields.io/github/issues/LieberInstitute/smokingMouse" alt="GitHub issues"></a></li>
<li><a href="https://github.com/LieberInstitute/smokingMouse/pulls" class="external-link"><img src="https://img.shields.io/github/issues-pr/LieberInstitute/smokingMouse" alt="GitHub pulls"></a></li>
<li><a href="https://lifecycle.r-lib.org/articles/stages.html#experimental" class="external-link"><img src="https://img.shields.io/badge/lifecycle-experimental-orange.svg" alt="Lifecycle: experimental"></a></li>
<li><a href="https://bioconductor.org/checkResults/release/data-experiment-LATEST/smokingMouse" class="external-link"><img src="http://www.bioconductor.org/shields/build/release/data-experiment/smokingMouse.svg" alt="Bioc release status"></a></li>
<li><a href="https://bioconductor.org/checkResults/devel/data-experiment-LATEST/smokingMouse" class="external-link"><img src="http://www.bioconductor.org/shields/build/devel/data-experiment/smokingMouse.svg" alt="Bioc devel status"></a></li>
<li><a href="http://bioconductor.org/packages/stats/bioc/smokingMouse/" class="external-link"><img src="https://bioconductor.org/shields/downloads/release/smokingMouse.svg" alt="Bioc downloads rank"></a></li>
<li><a href="https://support.bioconductor.org/tag/smokingMouse" class="external-link"><img src="https://bioconductor.org/shields/posts/smokingMouse.svg" alt="Bioc support"></a></li>
<li><a href="http://bioconductor.org/checkResults/devel/data-experiment-LATEST/smokingMouse/" class="external-link"><img src="https://bioconductor.org/shields/lastcommit/devel/data-experiment/smokingMouse.svg" alt="Bioc last commit"></a></li>
<li><a href="https://bioconductor.org/packages/release/bioc/html/smokingMouse.html#since" class="external-link"><img src="https://bioconductor.org/shields/dependencies/release/smokingMouse.svg" alt="Bioc dependencies"></a></li>
<li><a href="https://github.com/LieberInstitute/smokingMouse/actions/workflows/R-CMD-check-bioc.yaml" class="external-link"><img src="https://github.com/LieberInstitute/smokingMouse/actions/workflows/R-CMD-check-bioc.yaml/badge.svg" alt="R-CMD-check-bioc"></a></li>
</ul>
</div>

  </div>
</div>


      <footer><div class="copyright">
  <p></p>
<p>Developed by Daianna Gonzalez-Padilla.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.</p>
</div>

      </footer>
</div>






  </body>
</html>
